dotnetcoreVolunteer/appsettings.json

```diff
@@ -4,5 +4,6 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "awsConnectionString": "Data Source=database-1.cxl8e44xjnsm.us-east-2.rds.amazonaws.com,InitialCatalog=database-1,UserID=admin,Password=DABF2019" 
```

Comment:
This connection string looks wrong. The commas should be semicolons. This could be the reason the migrations failed

Reply:
Check the context first per Johnny then check the connection string if that doesn't work.

Reply:
Connection string is most likely incorrect but this should only affect things once the Update-Database command is run. Migrations are created locally so they should not be affected